tools/integration-tests/tester/tests/mana/mana_test.go

```diff
@@ -252,9 +254,10 @@ func TestApis(t *testing.T) {
 	// send funds to node 3 to trigger more consensus events.
 	_, err = peers[3].SendFaucetRequest(peers[3].Seed.Address(0).Address().Base58())
 	require.NoError(t, err)
+	time.Sleep(12 * time.Second)
 	resp9, err := peers[0].GoShimmerAPI.GetPastConsensusManaVector(timestampPast)
 	require.NoError(t, err)
-	assert.Equal(t, 3, len(resp9.Consensus)) //excluding node 3
+	assert.Equal(t, 4, len(resp9.Consensus))
```

Comment:
I thought this should be 3.
At `timestampPast` mana has been pledged to 3 nodes (0,1 and 2). `node3` shouldn't in the base mana vector.